ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/HttpResponseFactory.java

```diff
@@ -32,4 +36,28 @@ public void create(
     response.setChunked(false);
     response.end(Json.encodeToBuffer(body));
   }
+
+  public void successResponse(
+      final HttpServerResponse response, final JsonRpcRequestId id, final Object result) {
+    response.putHeader("Content", JSON);
+    response.setStatusCode(200);
+    response.setChunked(false);
+
+    final JsonRpcSuccessResponse body = new JsonRpcSuccessResponse(id, result);
+    response.end(Json.encodeToBuffer(body));
```

Comment:
Be good to remove some of the duplication between these methods. Looks like the only thing that differs is the status code and the response body that is returned.